devops/openshift/init-secrets.yml

```diff
@@ -206,3 +208,15 @@ parameters:
     required: true
     description: |
       Zeebe authorization server url.
+  - name: QUEUE_DASHBOARD_USER
+    description: Queue dashboard user
+    displayName: User for queue monitoring dashboard
+    generate: expression
```

Comment:
can we not generate the User? as per the description, i think its just a user display name.